ScoutSuite/output/data/html/report.html

```diff
@@ -152,23 +152,30 @@ <h4>
 
         </div> <!-- /container -->
 
-        <!-- Include all AWS services configs -->
-        <script src="scoutsuite-results/scoutsuite_results.js"></script>
-        <script src="scoutsuite-results/scoutsuite_exceptions.js"></script>
-
         <!-- Bootstrap core JavaScript, placed at the end of the document so the pages load faster -->
         <script src="inc-bootstrap/js/bootstrap.min.js"></script>
 
         <!-- Display dashbord -->
         <script>
-            load_metadata();
+            loadMetadata();
         </script>
 
+        <!-- Paging section -->
+        <div id="section_paging-div" style="display:flex;flex-direction:column">
+            <h2 id="section_paging-h2"></h2>
+            <div id="paging_buttons" style="margin:auto">
+                <button id="page_backward" type="button" class="btn btn-secondary" disabled><i class="fa fa-arrow-circle-left"></i> Back</button>
+                <!--button type="button" class="btn btn-secondary"><i class="fa fa fa-times-circle"></i> X</button-->
+                <button id="page_forward" type="button" class="btn btn-secondary">Next <i class="fa fa-arrow-circle-right"></i></button>                
+            </div>
+        </div>
```

Comment:
Can we move the inline css to a stylesheet?

Reply:
Ugh this is here because of I copied the CSV download buttons which already had inline CSS, what about we remove the inline CSS here when "we" will remove all the inline CSS? 😃